### PR TITLE
PR for 1.1.5 Bugfix Release

### PR DESCRIPTION
**Pull request by apeltzer**
Addressing the local executor stuff in multiqc.

### CODE REVIEW

File: main.nf [hunk 1]
```diff
@@ -374,4 +451,158 @@ process multiqc {
...
+    output_hf.withWriter { w -> w << email_html }
+    def output_tf = new File( output_d, "pipeline_report.txt" )
+    output_tf.withWriter { w -> w << email_txt }
+
+    c_reset = params.monochrome_logs ? '' : "\033[0m";
+    c_purple = params.monochrome_logs ? '' : "\033[0;35m";
+    c_green = params.monochrome_logs ? '' : "\033[0;32m";
+    c_red = params.monochrome_logs ? '' : "\033[0;31m";
+
+    if (workflow.stats.ignoredCountFmt > 0 && workflow.success) {
```
Comment (drpatelh) [inline main.nf]: Need to remove `Fmt` at the end of all of the `workflow.stats` variables in this block of code.

File: main.nf [hunk 2]
```diff
@@ -3,68 +3,110 @@
...
  #### Homepage / Documentation
  https://github.com/nf-core/hlatyping
+
  #### Authors
  Sven Fillinger sven1103 <sven.fillinger@qbic.uni-tuebingen.de> - https://github.com/sven1103>
  Christopher Mohr christopher-mohr <christopher.mohr@uni-tuebingen.de>
  Alexander Peltzer <alexander.peltzer@qbic.uni-tuebingen.de> - https://github.com/apeltzer
 ----------------------------------------------------------------------------------------
 */
+def readParamsFromJsonSettings() {
```
Comment (drpatelh) [inline main.nf]: Really like all of this i.e. reading the parameters in from the `parameters_settings.json`. Would be nice if we can find a way to move all of this code into another script to tidy things up. Just a comment :wink:

File: LICENSE [hunk 3]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright 2018, Sven Fillinger
```
[the remark this answers is not in the document]
Comment (apeltzer) [reply LICENSE]: We can change it as its basically me maintaining it nowadays. I'd have him and @christopher-mohr as contributors then?

[the remark this answers is not in the document]
Comment (drpatelh) [reply LICENSE]: I guess you should just keep all of the files with author information the same as you have in the `TEMPLATE` branch?

[the remark this answers is not in the document]
Comment (apeltzer) [reply LICENSE]: Hmm good point - then should probably also work on merging in the latest changes from TEMPLATE at some point. This pipeline was also used as a playground by @sven1103 for some of the stuff...

File: main.nf (hunk 2 above)
Comment (apeltzer) [reply main.nf]: Yes, that would be good - like a utility function 👍

**Pull request merged**